design/sketch.webidl

```diff
@@ -344,12 +344,12 @@ enum GPUCullMode {
 };
 
 dictionary GPURasterizationStateDescriptor {
-    GPUFrontFace frontFace;
-    GPUCullMode cullMode;
+    GPUFrontFace frontFace = "cw";
```

Comment:
I thought CCW was standard, but it appears Vulkan and OpenGL default to CCW, and D3D12 defaults to CW. So this default is basically arbitrary.

Reply:
Metal is CW